http_src/vue/page-hosts-ports-analysis.vue

```diff
@@ -293,13 +296,13 @@ async function update_dropdown_menus(is_application_selected, app, port) {
     port_list.value = [];
 
     res.rsp.forEach((item) => {
-        let name = item.l7_proto_name.split(".")[0];
-        ports.push({ label: `${item.srv_port}/${name} (${item.n_hosts})`, id: item.srv_port, application: name, num_hosts: item.n_hosts, vlan_id:item.vlan_id })
+        let name = item.l7_proto_name;
+        ports.push({ label: `${item.srv_port}/${name} (${item.n_hosts})`, id: item.srv_port, application: name, application_id: item.proto_id,num_hosts: item.n_hosts, vlan_id:item.vlan_id })
```

Comment:
In case of double application filter (master and app protocols) it should not work, please verify

Reply:
Fixed by e633628d9a8b79c4aa577a28f086912eba1553ca